Dangerfile

```diff
@@ -6,9 +6,23 @@ if (github.pr_body + github.pr_title).include?("WIP")
   warn("Pull Request is Work in Progress")
 end
 
-# Contributors should always provide a changelog when submitting a PR
+# Contributors should always provide a summary when submitting a PR
```

Comment:
Maybe it's time to ping Orta about it to start having Danger working again on fastlane?